torch_xla/core/xla_builder.py

```diff
@@ -828,57 +829,115 @@ def get_computation_hlo(computation):
   return torch_xla._XLAC._xla_computation_text(computation)
 
 
-class XlaComputation:
+def xla_computation_as_func(computation, name=None):
```

Comment:
Could we document this function: """Converts an `XlaComputation` object to a callable."""